docs/features/deserializing-events.md

```diff
@@ -5,6 +5,57 @@ layout: default
 
 ## Deserializing Events
 
+The `Arcus.EventGrid` package provides several ways to deserializing events.
+Following paragraphs describe each supported type of event.
+
+- [Deserializing EventGrid Events](#deserializing-eventgrid-events)
+- [Deserializing CloudEvent Events](#deserializing-cloudevent-events)
```

Comment:
Let's move this to second as we should aim to use CloudEvents over Event Grid native events